etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiOrderConverterHelper.java

```diff
@@ -0,0 +1,46 @@
+package gov.hhs.cdc.trustedintermediary.external.hapi;
+
+import java.util.List;
+import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.Coding;
+import org.hl7.fhir.r4.model.Patient;
+
+/** Helper class with transformation methods that take an order Bundle and modifies it */
+public class HapiOrderConverterHelper {
```

Comment:
Check the comments for the transformation objects. Thoughts on having the transformation logic as part of each object and having this class be a wrapper that couples with all the transformation objects?

Reply:
Agreed that we need refactoring and move logic between helper and transformations. As mentioned above, I propose to do it in a separate PR